### PR TITLE
Update SQL version

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/3040

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - [Change default username for client initialization to `root`](https://github.com/ballerina-platform/ballerina-standard-library/issues/2397)
 - [Fix incorrect retrieval of time types](https://github.com/ballerina-platform/ballerina-standard-library/issues/3023)
+- [Fix NullPointerException when retrieving record with default value](https://github.com/ballerina-platform/ballerina-standard-library/issues/2985)
```
Comment (niveathika) [inline changelog.md:14]: Lets update the heading, since these are now released.

Comment (kaneeldias) [reply changelog.md:21]: Fixed in [5f323b2](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/536/commits/5f323b274565bfef2134f91af4ae9e21bbe3be5e)

File: changelog.md [hunk 2]
```diff
@@ -10,7 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - [Change default username for client initialization to `root`](https://github.com/ballerina-platform/ballerina-standard-library/issues/2397)
+
+## [1.4.1] - 2022-06-27
+
+### Added
+
+### Changed
+- [Fix incorrect code snippet in SQL api docs](https://github.com/ballerina-platform/ballerina-standard-library/issues/2931)
```
Comment (niveathika) [inline changelog.md:17]:
```suggestion

```

ISnt this a duplicate?

Comment (kaneeldias) [reply changelog.md]: Fixed in [e2fc785](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/536/commits/e2fc785b2157e0f5e2ed131c4d47665f14da010b)

**Review by niveathika — APPROVED** [review]

**Pull request closed without merging**